tinker/templates/admin/redirects/home.html

```diff
@@ -164,12 +164,35 @@
                     input.val("/" + value); // add in a slash
                     value = input.val();
                 }
+                if (value[0] == '/' && value[1] =='/'){
+                    input.val(input.val().replace('//','/')); // eliminates a lead double slash
```

Comment:
technically, this still allows a triple slash to be changed to a double slash - which won't be changed to a single slash

Reply:
Maybe do a replace all? And maybe do a string check to see if // is in the string? Or, you could consider just running it everytime regardless.

Reply:
It now checks for more than two slashes and gets rid of all of them